lib/availability_resolver.js

```diff
@@ -162,6 +146,12 @@ class AvailabilityResolver {
             item.requestable = [DeliveryLocationsResolver.eddRequestableByOnSiteCriteria(item)]
```

Comment:
This should be changed to set `eddRequestable` instead of `requestable`. (And as a plain boolean, not an array.)